src/verifier_test.cc

```diff
@@ -250,7 +250,7 @@ TEST_F(VerifierTest, ProbeFrameBufferUInt32) {
                             4 * static_cast<uint32_t>(sizeof(uint32_t)), 1, 1,
                             static_cast<const void*>(&frame_buffer));
   EXPECT_TRUE(r.IsSuccess());
-  EXPECT_STREQ("", r.Error());
+  EXPECT_STREQ("", r.Error().c_str());
```

Comment:
This shouldn't be needed. Use:

```
EXPECT_EQ("", r.Error());
```
and it should do string comparisons.

Reply:
Done